### ISSUE TITLE
Lubuntu 20.04 64-bit -- can't install btsync-gui

### ISSUE THREAD

**Issue by JimH44** [Silvenga/btsync-deb#21]
Hello Silvenga, I love your package, and have been using it on Lubuntu 18.04, 32-bit, but I can't install it on Lubuntu 20.04.3, 64-bit.

Here's what happens:

    sudo apt-get install btsync-gui

but it produces this error:

     btsync-gui : Depends: btsync-gui-gtk but it is not going to be installed

I tried to install _btsync-gui-gtk_ the same way, but got this error:

     btsync-gui-gtk : Depends: python-requests but it is not installable
                  Depends: python-qrencode but it is not installable

I tried to install _python-requests_ using _pip_ but was not able to.

Would you have time to bundle a 64-bit version of _btsync-gui_ that contains these dependencies?

Thanks for considering this request.

**Comment by Silvenga** [comment]
I don't think this is a bitness problem, Ubuntu has the 32-bit compatibility layer (by default?) - can you try and installing `python-qrencode` manually?

I suspect this may be related to Python3.

**Comment by JimH44** [comment]
Thanks for the comment.
I had already tried

    sudo apt-get update
    sudo apt-get install python-qrencode

and got this error:

    Package python-qrencode is not available, but is referred to by another package.
    This may mean that the package is missing, has been obsoleted, or
    is only available from another source

E: Package 'python-qrencode' has no installation candidate

I think the Ubuntu repositories have moved on to python3, since 2 is no longer being supported. So I tried

    sudo apt-get install python3-qrencode

and got success:

    python3-qrencode is already the newest version (1.2-5build1).
    0 to upgrade, 0 to newly install, 0 to remove and 0 not to upgrade.

Would you have time to release a python3 version with the 64-bit binaries in it?  :-)

**Comment by Silvenga** [comment]
Unformatately, we're going to see a lot of Python 2 apps start to die off over the next couple year, this project will likely be one of them. I'm not a python dev, so I can't do the upgrade.

I come from .NET, and .NET can run 20 year old compiled binaries in the modern runtimes - so I'm used to (or spoiled of?) long term binary and behavior compatibility. This move by Python will fragment the ecosystem for many years to come, kind of sad.

**Comment by JimH44** [comment]
Yes, python is disappointing that way. Thanks for your part in packaging up the deb files.

Would tuxpoldo be interested in converting it to python3?
